src/install.py

```diff
@@ -335,7 +335,7 @@ def parse_config(self):
         if d.env_setup == 'conda-basic':
             d.conda_envs = [self.settings.conda['framework_env'], 'NCL_base']
             d.environment_manager = "Conda"
-            d.pods = ["Wheeler_Kiladis", "EOF_500hPa", "MJO_suite", "MJO_teleconnection"]
+            d.pods = ["Wheeler_Kiladis", "EOF_500hPa", "MJO_suite", "MJO_teleconnection", "flow_dep_diag"]
```

Comment:
You shouldn't need to modify anything in the src files (unless @tsjackson-noaa or @jkrasting recommended it). The basic setup applies only to the 1st-gen PODs that run NCL.